From bbcf83fc76b8e581ce8764e75d6998d3b7e5d404 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Mon, 6 Nov 2023 18:32:23 +0800
Subject: [PATCH 1/4] amlbian: Set the logo loading path [1/1]

PD#SWPL-144989

Problem:
No logo displayed during the uboot stage.

Solution:
Set the correct logo loading path.

Verify:
local

Change-Id: I11ad854d9d159ed63b3f9768149a3257a4769ea1
Signed-off-by: yi.liu <yi.liu@amlogic.com>
---
 board/amlogic/configs/g12a_u200_debian_v1.h | 2 +-
 board/amlogic/configs/g12b_w400_debian_v1.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/amlogic/configs/g12a_u200_debian_v1.h b/board/amlogic/configs/g12a_u200_debian_v1.h
index bc55d71d9c7..ecfb97dc587 100644
--- a/board/amlogic/configs/g12a_u200_debian_v1.h
+++ b/board/amlogic/configs/g12a_u200_debian_v1.h
@@ -330,7 +330,7 @@
 		"fi;"\
 		"\0"\
 	"load_bmp_logo="\
-		"if load mmc 1:3 ${loadaddr} /usr/share/amlbian/logo/logo.bmp; then "\
+		"if load mmc 1:6 ${loadaddr} /usr/share/amlbian/logo/logo.bmp; then "\
 			"bmp display ${loadaddr};"\
 			"bmp scale;"\
 		"fi;"\
diff --git a/board/amlogic/configs/g12b_w400_debian_v1.h b/board/amlogic/configs/g12b_w400_debian_v1.h
index 5052db2df61..1c731f225b4 100644
--- a/board/amlogic/configs/g12b_w400_debian_v1.h
+++ b/board/amlogic/configs/g12b_w400_debian_v1.h
@@ -310,7 +310,7 @@
 			"fi;"\
 			"\0"\
 		"load_bmp_logo="\
-			"if load mmc 1:3 ${loadaddr} /usr/share/amlbian/logo/logo.bmp; then "\
+			"if load mmc 1:6 ${loadaddr} /usr/share/amlbian/logo/logo.bmp; then "\
 				"bmp display ${loadaddr};"\
 				"bmp scale;"\
 			"fi;"\
-- 
2.25.1

